HW7/PismakArchip/7.3.py

```diff
@@ -0,0 +1,12 @@
+def character_count(s):
```

Comment:
good job
but look at this version of code
def count_symbols_in_row(word):
    
    result = {}
    
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result